user-service/src/main/java/uk/gov/gchq/palisade/service/user/service/UserService.java

```diff
@@ -30,13 +31,13 @@
  */
 public interface UserService extends Service {
     /**
-     * Look up a user by their ID. The request contains the {@link uk.gov.gchq.palisade.UserId} to lookup from the
-     * {@link UserService}. If the requested {@link uk.gov.gchq.palisade.UserId} doesn't exist in this {@link
+     * Look up a user by their ID. The request contains the {@link UserId} to lookup from the
```

Comment:
I prefer this as it was before e.g full qualified class in the javadoc rather than importing it when it is only needed in the javadoc

Reply:
The UserId is the argument to the function, will be imported ayway